.github/workflows/update-bidi-types.yml

```
@@ -0,0 +1,65 @@
+# This workflow will update the WebdriverBidi types.
+
+name: Update WebdriverBidi types
```

```suggestion
name: Update WebDriverBidi types
```

(very small nit - but to align with your own addition below)